pfcpiface/parse-far_test.go

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022-present Open Networking Foundation
+
+package main
+
+import (
+	"net"
+	"testing"
+
+	"github.com/omec-project/upf-epc/test/integration"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/wmnsk/go-pfcp/ie"
+)
+
+type testCase struct {
+	input       *ie.IE
+	op          operation
+	description string
+}
+
+func TestParseFAR(t *testing.T) {
+	createOp, updateOp := create, update
+
+	for _, scenario := range []testCase{
+		{
+			op:          createOp,
+			input:       integration.NewUplinkFAR(integration.IEMethod(createOp), 999, ActionDrop),
+			description: "Valid Uplink FAR input with create operation",
+		},
+		{
+			op: updateOp,
+			input: integration.NewDownlinkFAR(integration.IEMethod(updateOp),
+				1,
+				ActionForward,
+				100,
+				"10.0.0.1",
+			),
+			description: "Valid Downlink FAR input with update operation",
+		},
+	} {
+		t.Run(scenario.description, func(t *testing.T) {
+			mockFar := far{}
```

Comment:
Do we print `scenario.description` anywhere?

Reply:
yes (see screenshot)

![image](https://user-images.githubusercontent.com/26747011/151565921-8ab1550e-9fd0-47cc-a46a-fae0153a5bd1.png)